README.md

```diff
@@ -1,12 +1,299 @@
+| ------------------------------- | ------------------------------------------------------------------------------------------------------------ | --------------------------- | ------------------------ | ------------------------------------ |
+| log.console.level               | Console logger level                                                                                         | info                        | info, debug, error, warn | CRON_LOG_CONSOLE_LEVEL               |
+| log.file                        | Enables logging on file (location: /var/log/influxdb-storer-logs-%DATE%.log)                                 | false                       | boolean                  | CRON_LOG_FILE                        |
+| log.file.level                  | Log level to log on files                                                                                    | info                        | string                   | CRON_LOG_FILE_LEVEL                  |
+| log.verbose                     | Whether to enable logger verbosity or not                                                                    | false                       | boolean                  | CRON_LOG_VERBOSE                     |
+| healthChecker.kafka.interval.ms | Specific how often it is to check if it is possible to communicate with the *kafka* service in milliseconds. | 30000                       | integer                  | CRON_HEALTHCHECKER_KAFKA_INTERVAL_MS |
+| dojot.subjects.tenancy          | Suffix for dojot topic that receives exclusion and tenant creation events.                                   | dojot.tenancy               | string                   | CRON_DOJOT.SUBJECTS.TENANCY          |
+| dojot.subjects.devices          | Suffix for the dojot topic that sends data to devices and receives device lifecycle events.                  | dojot.device-manager.device | string                   | CRON_DOJOT.SUBJECTS.DEVICES          |
+| dojot.subjects.deviceData       | Suffix for the dojot topic that receives data from devices.                                                  | device-data                 | string                   | CRON_DOJOT.SUBJECTS.DEVICEDATA       |
+
+#### MongoDB Configurations
+
+| Key                          | Purpose                                                                                                   | Default Value | Valid Values            | Environment variable              |
+| ---------------------------- | --------------------------------------------------------------------------------------------------------- | ------------- | ----------------------- | --------------------------------- |
+| db.mongodb.url               | Url and port for connection to the dataBase                                                               | string        | mongodb://mongodb:27017 | CRON_DB_MONGODB_URL               |
```

Comment:
Don't use capital letters in the labels of configuration files, lib doesn't work well with that. Use tiny and dots.

Reply:
Change for db.options.connectTimeoutMS or dboptions.connectTimeoutMS